virtual-server/examples/terraform/vs.tf

```diff
@@ -0,0 +1,57 @@
+resource "kubernetes_manifest" "virtualdesktop" {
+  provider = kubernetes-alpha
+
+  manifest = {
+    "apiVersion" = "virtualservers.coreweave.com/v1alpha1"
+    "kind"       = "VirtualServer"
+    "metadata" = {
+      "name"      = var.vs_name
+      "namespace" = var.user_namespace
+    }
+    "spec" = {
+      "initializeRunning" = var.vs_running
+      "network" = {
```

Comment:
Can we add in attachLoadBalancerIP here as well. It gets the Service IP directly attached to the VM (it becomes the Pod IP). It's a pretty common use case so good for completeness.